alert-common/src/main/java/com/synopsys/integration/alert/common/channel/issuetracker/message/IssueContentModel.java

```diff
@@ -0,0 +1,72 @@
+/**
+ * alert-common
+ *
+ * Copyright (c) 2020 Synopsys, Inc.
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package com.synopsys.integration.alert.common.channel.issuetracker.message;
+
+import java.util.Collection;
+import java.util.LinkedList;
+
+public class IssueContentModel {
```

Comment:
We should see if this will work for the Azure work items

Reply:
This may need to change but as we implement the Azure channel content we will be able to make a better determination on refactoring these objects in the issuetracker package.